internal/devserver/ts_devserver.bzl

```diff
@@ -47,6 +47,24 @@ def _ts_devserver(ctx):
   else:
     workspace_name = ctx.workspace_name
 
+  script_files = depset()
+  script_files += ctx.files._requirejs_script
+  script_files += ctx.files.scripts
+  ctx.actions.write(ctx.outputs.scripts_manifest, "".join([
+    # TODO: change to $(rootpath) after upgrading users to Bazel 0.8
+    expand_path_into_runfiles(ctx, f.path) + "\n" for f in script_files
+  ]))
+
+  livereload_script_path = expand_path_into_runfiles(ctx, ctx.files._livereload_script[0].path)
+
+  devserver_runfiles = [ctx.executable._devserver, \
```

Comment:
why trailing backslashes?

Reply:
```ERROR: /private/var/tmp/_bazel_greg/5261fbd4ffdbd6b363290b88bb471f89/external/build_bazel_rules_typescript/internal/devserver/ts_devserver.bzl:62:36: syntax error at 'newline': expected expression.
ERROR: /private/var/tmp/_bazel_greg/5261fbd4ffdbd6b363290b88bb471f89/external/build_bazel_rules_typescript/internal/devserver/ts_devserver.bzl:103:1: syntax error at 'outdent': expected expression.
ERROR: error loading package '': Extension 'internal/devserver/ts_devserver.bzl' has errors.```

Reply:
um, that is strange, what if you start with newline after the `[`?

Reply:
nope. skylark doesn't seem to like multiline expressions without `\`

Reply:
this works
```
  devserver_runfiles = [
    ctx.executable._devserver,
    ctx.outputs.manifest,
    ctx.outputs.scripts_manifest]
  devserver_runfiles += ctx.files.static_files
  devserver_runfiles += ctx.files._livereload_script
  devserver_runfiles += ctx.files._requirejs_script
  devserver_runfiles += ctx.files.scripts
```

Reply:
maybe a bit easier to read